File: PHP/PrimeNumber.php
/* Prime Number Finder */    
<?php  
    $count = 0;  
    $num = 2;  
    while ($count < 15 ){  
      $div_count=0;  
      for ( $i=1; $i<=$num; $i++){  
          if (($num%$i)==0){  
            $div_count++;  
             }  
          }
      if ($div_count<3){  
        echo $num." , ";  
        $count=$count+1;  
        }  
    $num=$num+1;  
    }  
?>  

File: Python/prime-number.py
if(n > 1):
	for k in range(2, int(sqrt(n)) + 1):
		if (n % k == 0):
			flag = 1
		break
	if (flag == 0):
		print(n," is a Prime Number!")
	else:
		print(n," is Not a Prime Number!")
else:
	print(n," is Not a Prime Number!")
